mindjunkies/lecture/admin.py

```
@@ -23,3 +22,11 @@ class LecturePDFAdmin(ModelAdmin):
 class LectureVideoAdmin(ModelAdmin):
     list_display = ("video_title", "lecture", "video_file", "status", "is_running")
     search_fields = ("lecture__title",)
+
+
+
+@admin.register(LectureCompletion)
+class LectureCompletionAdmin(ModelAdmin):
+    list_display = ("user", "lecture", "completed_at")
+    search_fields = ("user__username", "lecture__title")
+    list_filter = ("completed_at",)
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)